Non-regular file removal test fails on GNU/Hurd

While building zstd on GNU/Hurd the following test fail:
> skip>
> 
>test : file removal
>
> Read : 0 / 0 MB ==> 32% 
>                                                                                
> tmp                  : 32.42%   ( 65537 =>  21246 bytes, tmp.zst) 
> 
> tmp.zst              : 0 MB...     
>                                                                                
> tmp.zst             : 65537 bytes 
> test : should quietly not remove non-regular file
> Makefile:306: recipe for target 'zstd-playTests' failed
> make[3]: *** [zstd-playTests] Error 1

The complete build log lives [here](https://buildd.debian.org/status/fetch.php?pkg=libzstd&arch=hurd-i386&ver=1.3.4%2Bdfsg-2&stamp=1525276222&raw=0).
Any idea what might be wrong there? Thanks!

--- a/tests/playTests.sh
+++ b/tests/playTests.sh
@@ -176,7 +176,7 @@ $ECHO hello > tmp
 $ZSTD tmp -f -o "$DEVDEVICE" 2>tmplog > "$INTOVOID"
 grep -v "Refusing to remove non-regular file" tmplog
 rm -f tmplog
-$ZSTD tmp -f -o "$INTONULL" 2>&1 | grep -v "Refusing to remove non-regular file"
+$ZSTD tmp -f -o "$INTOVOID" 2>&1 | grep -v "Refusing to remove non-regular file"
 $ECHO "test : --rm on stdin"
 $ECHO a | $ZSTD --rm > $INTOVOID   # --rm should remain silent
 rm tmp

